### PR TITLE
Some changes to logging and initialization logic.

### PR DESCRIPTION
**Pull request by mdwelsh**
These changes are meant to address a couple of startup issues:

* There is a separate 'startAudio()' step that can be optionally called before 'start()' to get mic permissions before the peer starts speaking.
* Added a 'audioContext.resume()' in '.startAudio()' to fix issues where the AudioContext was initialized outside of a click handler and needed to be resumed.
* Added a bit of extra logging to help with debugging.
* There are some minor style cleanups here mostly to make lint happy.

### CODE REVIEW

File: packages/fixie/src/voice.ts [hunk 1]
```diff
@@ -51,65 +52,110 @@ export class StreamAnalyzer {
...
-  private audioElement = new Audio();
-  private textEncoder = new TextEncoder();
-  private textDecoder = new TextDecoder();
+  private readonly audioContext = new AudioContext();
+  private readonly audioElement = new Audio();
+  private readonly textEncoder = new TextEncoder();
+  private readonly textDecoder = new TextDecoder();
   private _state = VoiceSessionState.IDLE;
   private socket?: WebSocket;
+  private _audioStarted = false;
```
Comment (juberti) [inline packages/fixie/src/voice.ts:61]: remove underscores, not used in TS style

File: packages/fixie/src/voice.ts [hunk 2]
```diff
@@ -123,31 +169,48 @@ export class VoiceSession {
...
+
   private maybePublishLocalAudio() {
-    if (this.room && this.room.state == 'connected' && this.localAudioTrack) {
-      console.log(`[chat] publishing local audio track`);
+    if (this._started && this.room && this.room.state == 'connected' && this.localAudioTrack) {
+      console.log('[voiceSession] publishing local audio track');
       const opts = { name: 'audio', simulcast: false, source: Track.Source.Microphone };
       this.room.localParticipant.publishTrack(this.localAudioTrack, opts);
+    } else {
+      console.warn(
```
Comment (juberti) [inline packages/fixie/src/voice.ts:200]: this is normal operation, shouldn't be a `warn()`

**Comment by mdwelsh** [comment]
PTAL

File: packages/fixie/src/voice.ts [hunk 3]
```diff
@@ -51,65 +52,110 @@ export class StreamAnalyzer {
...
+  get outputAnalyzer(): AnalyserNode | undefined {
     return this.outAnalyzer?.analyzer;
   }
-  warmup() {
-    const url = this.params?.webrtcUrl || 'wss://wsapi.fixie.ai';
+
+  /** Warm up the VoiceSession by making network connections. This is called automatically when the VoiceSession object is created. */
+  warmup(): void {
+    console.log('[voiceSession] warming up');
+    this._audioStarted = false;
```
[the remark this answers is not in the document]
Comment (juberti) [reply packages/fixie/src/voice.ts]: did you get a chance to look at this one?

**Pull request merged**